### DISCUSSION TITLE
Regarding Audit events

### DISCUSSION THREAD

**Discussion by nitishfy** [checkstyle/checkstyle#12299] [Q&A] [answered]
While going through the documentation, there is a term 'Audi events' which is being used very frequently. May I know what do we actually mean by an Audit event?

**Comment by rnveach** [comment, marked as answer]
We have a class called AuditEvent.
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/api/AuditEvent.java

This is how we pass things to filters and listeners letting them know of a violation or when a certain event is occuring (starting file, etc...).

**Comment by romani** [comment]
It make sense to extend doc to put link to class to avoid such questions in future. Please send PR.

**Comment by nitishfy** [comment]
Alright! I'd raise a PR for this issue..
